test/sqllogictest/audit_log.slt

```diff
@@ -107,14 +107,14 @@ SELECT id, event_type, object_type, details, user FROM mz_audit_events ORDER BY
 25  create  index  {"database":"materialize","id":"u6","item":"renamed_primary_idx","schema":"public"}  materialize
 26  drop  index  {"database":"materialize","id":"u6","item":"renamed_primary_idx","schema":"public"}  materialize
 27  drop  view  {"database":"materialize","id":"u2","item":"renamed","schema":"public"}  materialize
-28  create  source  {"database":"materialize","id":"u7","item":"s","schema":"public","size":"1"}  materialize
+28  create  source  {"database":"materialize","id":"u7","item":"s","schema":"public","size":"1","type":"load-generator"}  materialize
 29  alter  source  {"database":"materialize","id":"u7","item":"s","new_size":"2","old_size":"1","schema":"public"}  materialize
 30  drop  source  {"database":"materialize","id":"u7","item":"s","schema":"public"}  materialize
-31  create  source  {"database":"materialize","id":"u8","item":"accounts","schema":"public","size":null}  materialize
-32  create  source  {"database":"materialize","id":"u9","item":"auctions","schema":"public","size":null}  materialize
-33  create  source  {"database":"materialize","id":"u10","item":"bids","schema":"public","size":null}  materialize
-34  create  source  {"database":"materialize","id":"u11","item":"organizations","schema":"public","size":null}  materialize
-35  create  source  {"database":"materialize","id":"u12","item":"users","schema":"public","size":null}  materialize
-36  create  source  {"database":"materialize","id":"u13","item":"multiplex","schema":"public","size":"1"}  materialize
+31  create  source  {"database":"materialize","id":"u8","item":"accounts","schema":"public","size":null,"type":"subsource"}  materialize
+32  create  source  {"database":"materialize","id":"u9","item":"auctions","schema":"public","size":null,"type":"subsource"}  materialize
+33  create  source  {"database":"materialize","id":"u10","item":"bids","schema":"public","size":null,"type":"subsource"}  materialize
+34  create  source  {"database":"materialize","id":"u11","item":"organizations","schema":"public","size":null,"type":"subsource"}  materialize
+35  create  source  {"database":"materialize","id":"u12","item":"users","schema":"public","size":null,"type":"subsource"}  materialize
+36  create  source  {"database":"materialize","id":"u13","item":"multiplex","schema":"public","size":"1","type":"load-generator"}  materialize
```

Comment:
I'm concerned now that this file doesn't have kafka or postgres sources here. Can one or both of those be added?

Reply:
I added a couple of tests to testdrive because it's not possible to make real sources in slt. I tested S3 and Kafka but left our Postgres because apparently the testdrive folder doesn't support Postgres sources.